koku/koku/test_trino_db_utils.py

```diff
@@ -105,6 +134,44 @@ def cursor(self):
 select x from y;
 select a from b;
 """
-        with self.assertRaises(trino_db.TrinoStatementExecError):
+        with self.assertRaises(ValueError):
```

Comment:
nitpick question:   Curious why this is now a `ValueError` from `TrinoStatementExecError`?

Reply:
I was hoping someone would ask about that. I changed the behavior in `executescript()` to only raise `TrinoStatementExecError` if a` TrinoQueryError` was raised since that is now a required parameter of `TrinoStatementExecError. Otherwise, just raise the original exception.

Reply:
Thanks for the explanation 👍🏾 , and I see the `ValueError` raised by `FakerFakeTrinoConn`